pkg/processor/build/runtime/nodejs/docker/onbuild/Dockerfile

```diff
@@ -23,3 +23,4 @@ FROM scratch
 
 COPY --from=processor /home/nuclio/bin/processor /home/nuclio/bin/processor
 COPY pkg/processor/runtime/nodejs/wrapper.js /home/nuclio/bin/wrapper.js
+COPY pkg/processor/build/runtime/nodejs/docker/_nuclio_install.sh /home/nuclio/bin
```

Comment:
Is this file really necessary? Can't we just `cd <> && npm install` in the Dockerfile?

Reply:
We need `cd $(dirname <path>)` which I'm not sure how Docker will handle and it makes the command more complicated. I prefer to place all the logic in a file and keep the command simple.

Your call...

Reply:
It's hard for me to justify a 13 line script that only does `cd $(dirname path)`. It makes you think you're missing something (which is what I thought reading the script)... I prefer to have this inline.

Reply:
Done